docs/source/gallery/model_comparison/plot_compare.py

```diff
@@ -0,0 +1,34 @@
+"""
+(gallery_forest_pp_obs)=
+# Posterior predictive and observations forest plot
+
+Overlay of forest plot for the posterior predictive samples and the actual observations
+
+---
+
+:::{seealso}
+API Documentation: {func}`~arviz_plots.plot_forest`
+
+Other gallery examples using `plot_forest`: {ref}`gallery_forest`, {ref}`gallery_forest_shade`
+:::
+"""
+from importlib import import_module
+
+from arviz_base import load_arviz_data
+
+import arviz_plots as azp
+
+azp.style.use("arviz-clean")
+
+backend="none"  # change to preferred backend
+
+cmp_df = pd.DataFrame({"elpd_loo": [-4.575778, -14.309050, -16], 
```

Comment:
Not sure about this

Reply:
I think it is ok, it will be faster and it also probably looks nicer with 3 than with 2 models